src/components/molecules/map/GoogleMapsLayer.tsx

```diff
@@ -1,12 +1,20 @@
 import React, { FC } from 'react';
 import ReactLeafletGoogleLayer from 'react-leaflet-google-layer';
+import { useParams } from 'react-router-dom';
 import { MAP_STYLE } from 'style/map.style';
 import { mapApiKey } from 'utils/utils';
 
+interface lng {
+  lng: string;
+}
+
 const GoogleMapsLayer: FC = ({ children }) => {
+  const { lng }: lng = useParams();
+  const selectedLang: string = lng ? lng : 'he';
```

Comment:
can we use `lng || 'he'` instead?
should we prefer to import a constant of `locales.hebrew`, is it the convention in the app or should we create one?

Reply:
I edited the code as you suggested, thanks.
Not sure about the `locals`, I need someone to tell me what to do about this @yuvalbl @SejoB